headers/xdp/prog_dispatcher.h

```diff
@@ -5,7 +5,15 @@
 
 #include <linux/types.h>
 
-#define XDP_METADATA_SECTION "xdp_metadata"
+#define XDP_METADATA_SECTION_ORIG "xdp_metadata"
+#define XDP_METADATA_SECTION_NEW ".data.xdp_metadata"
```

Comment:
Surely this doesn't need to be renamed since this is not a program section name - therefore libbpf should ignore it.
Otherwise this is a libbpf bug IMHO.